src/js/translations/en-US.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
@weblancaster can you amend our Contributing guidelines to add something around using the `FormattedMessage` component for any piece of copy?

Please include the naming scheme for IDs.